### ISSUE TITLE
Pass optimizer object

### ISSUE THREAD

**Issue by tadejsv** [jina-ai/finetuner#194]
Instead of name/kwargs - keep the default, also an argument for learning rate
